packages/blade/src/utils/isReactNative/index.ts

```diff
@@ -0,0 +1,3 @@
+export { isReactNative } from './isReactNative';
+export { Platform } from './platform';
```

Comment:
Can you move all of these into `utils/platform.ts`? `isReactNative` doesn't seem like the right file to import these utils for Platform from

Reply:
Done.